### ISSUE TITLE
Mirroring tags between github repos

### ISSUE THREAD

**Issue by colincadams** [google/copybara#81]
It isn't clear to me how to mirror tags between two github repositories. I current workflow for exporting source looks like this:
```python
core.workflow(
    name = "exportSourceToPublic",
    origin = git.github_origin(
        url = privateUrl,
        ref = "master",
    ),
    destination = git.github_destination(
        url = publicUrl,
        fetch = "master",
        push = "master",
    ),
    origin_files = filesToPublish,
    mode = "ITERATIVE",
    authoring = authoring.pass_thru(defaultAuthor),
    transformations = [
        metadata.restore_author(),
    ],
)
```

All of our tags are for commits on `master`. I assumed that I could simply run this with `origin.ref = "refs/tags/<tag>"` and `destination.push = "refs/tags/<tag>"`. When I do this I get this output:
```
Task: Git Origin: Initializing local repo
WARN: No new changes to import for resolved ref: 1809e6d8aa53af7fa30146277f9b8740ab38771c
```

This makes sense as the commit already exists on `destination:master`, but I would still expect it to push it to the tag. Am I holding it wrong / is there a better way to handle this use case? Or is it a bug and it should still push to the tag?

**Comment by mikelalcon** [comment]
Yeah, that is not going to work. Even if it would (you could force it with --last-rev) it would create a new commit, not reuse the existing one in the external repo. The best way is to create a the tag manually from the external commit (or create a wrapper script0
